src/core/qgsaggregatecalculator.h

```diff
@@ -171,10 +171,11 @@ class CORE_EXPORT QgsAggregateCalculator
      * If an expression is used, then the context parameter must be set.
      * \param context expression context for evaluating expressions
      * \param ok if specified, will be set to TRUE if aggregate calculation was successful
+     * \param feedback optional feedback argument for early cancellation (since QGIS 3.22)
```

Comment:
Maybe add a line about this taking precedence over the feedback from the context.